azure/scope/cluster.go

```diff
@@ -691,6 +710,7 @@ func (s *ClusterScope) PatchObject(ctx context.Context) error {
 			infrav1.BastionHostReadyCondition,
 			infrav1.VNetReadyCondition,
 			infrav1.SubnetsReadyCondition,
+			infrav1.PrivateDNSReadyCondition,
```

Comment:
In [this PR](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2093/files) we removed the SetSummary part. When you rebase, you can remove this line.